source/quick-start.txt

```diff
@@ -7,193 +7,202 @@ Quick Start
-file in your project directory, you may need to set your system to
-display hidden files. Once your ``.env`` is open, replace
-``<connection_uri>`` with your connection string from Atlas. Ensure you
-update ``<username>`` and ``<password>`` in your connection string.
+Log into your Atlas account and navigate to the **Clusters** section and
+click the **Connect** button for the cluster in which you want to connect
+as shown below.
 
-Connect to your Cluster
------------------------
+.. figure:: /includes/figures/atlas_connection_select_cluster.png
 
-For the final step, we'll write some code that tests your environment's
-connection to your instance of MongoDB.
+Follow the instructions in the dialog to whitelist your IP address and create
+a new MongoDB user with **atlasAdmin** permissions. Then proceed to
+**Choose a connection method**.
 
-Create a file in your project directory called ``index.js``:
+From the list of connection methods, select **Connect Your Application** and
+select the Node.js driver. Select the "Connection String Only" tab and click
+the **Copy** button to copy the *connection string* to your clipboard as
+shown below.
 
-.. code-block:: sh
+.. figure:: /includes/figures/atlas_connection_copy_string.png
 
-  touch index.js
+Save your connection string to a safe location that you can access in the
+next step.
 
-Open this file and paste in the following code:
+Next, create a file to contain your application called ``index.js`` in your
+``node_quickstart`` directory. Add the following code, replacing the ``uri``
+variable with your connection string. Make sure to replace the "<password>"
+section of the connection string with the password you created for your user
+that has **atlasAdmin** permissions.
 
 .. code-block:: js
 
   const { MongoClient } = require("mongodb");
-  require("dotenv").config();
 
-  if (!process.env.MONGODB_URI) {
-    throw Error("MONGODB_URI not defined in a .env file. Did you set it up?");
-  }
+  // Replace the uri string with your MongoDB deployment's connection string.
+  const uri =
+    "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority&useUnifiedTopology=true";
 
-  const client = new MongoClient(process.env.MONGODB_URI, {
-    useUnifiedTopology: true
-  });
+  const client = new MongoClient(uri);
 
   async function run() {
     try {
-      // Connect the client to the server
       await client.connect();
-      // Establish and verify connection
-      await client.db("admin").command({ ping: 1 });
-      console.log("Successfully connected to Atlas.");
-      // your code after this comment
 
-      // your code before this comment
+      const database = client.db('sample_mflix');
+      const collection = database.collection('movies');
+
+      // Query for a movie that has the title 'Back to the Future'
+      const query = { title: 'Back to the Future' };
+      const movie = await collection.findOne(query);
+
+      console.log(movie);
```

Comment:
console.log(collection.findOne({})?

Or

doc = collection.findOne({});
console.log(doc);

? 

I'm flexible here honestly, just thought a query might add unnecessary confusion.

Reply:
I would strongly prefer to keep the same patterns across all our CRUD operations.
The reason I provided a query is so that the user has control of what they are querying for as well as to match the sample output. This might also encourage experimentation with different queries whereas findOne without a query is more difficult for a user to figure out how to modify.

Reply:
Sounds good.